natgeo/mortar
# [SYTO-1099] Added a mixin for verlag and haas with the new rules

To test:

- Look for Verlag text and verify that it contains:
```css
letter-spacing: 0.2em;
text-transform: uppercase;
word-spacing: 0.05em;
```
- Look for Hass text and verify that it contains:
```css
letter-spacing: 0.04em;
```
